src/Components/Mobile_filter/Mobile_filter.jsx

```diff
@@ -0,0 +1,240 @@
+              <span className="label fnt-1-2">Studio-Equipments</span>
+            </label>
+          </div>
+          <div className="spacer"></div>
+          <div className="flex-center-column ali-st gap-0-8">
+            <div className="fnt-1 fnt-w-600">Ratings</div>
+            <label
+              className="radio flex-center-row gap-0-5"
+              onChange={() => {
+                setFilterState({
+                  type: "rating",
+                  paylod: 4,
+                });
+              }}
+            >
+              <input
+                type="radio"
+                name="ratings"
+                id="my_radio"
+                className="radio__input"
+                checked={filter_state.rating === 4}
+              />
+              <div className="radio__radio pad-2px"></div>4 Stars & Above
+            </label>
+            <label
+              className="radio flex-center-row gap-0-5"
+              onChange={() => {
+                setFilterState({
+                  type: "rating",
+                  paylod: 3,
+                });
+              }}
+            >
+              <input
+                type="radio"
+                name="ratings"
+                id="my_radio"
+                className="radio__input"
+                checked={filter_state.rating === 3}
+              />
+              <div className="radio__radio pad-2px"></div>3 Stars & Above
+            </label>
+            <label
+              className="radio flex-center-row gap-0-5"
+              onChange={() => {
+                setFilterState({
+                  type: "rating",
+                  paylod: 2,
+                });
+              }}
+            >
+              <input
+                type="radio"
+                name="ratings"
+                id="my_radio"
+                className="radio__input"
+                checked={filter_state.rating === 2}
+              />
+              <div className="radio__radio pad-2px"></div>2 Stars & Above
+            </label>
+            <label
+              className="radio flex-center-row gap-0-5"
+              onChange={() => {
+                setFilterState({
+                  type: "rating",
+                  paylod: 1,
+                });
+              }}
+            >
+              <input
+                type="radio"
+                name="ratings"
+                id="my_radio"
+                className="radio__input"
+                checked={filter_state.rating === 1}
+              />
+              <div className="radio__radio pad-2px"></div>1 Stars & Above
+            </label>
+          </div>
+          <div className="spacer"></div>
+          <a
+            onClick={() => {
+              setshow_filter((state) => !state);
+            }}
+            className="apply flex-center-column fnt-1-5 pad-tb-1"
+          >
+            Close Filter
+          </a>
+        </div>
+      )}
+    </div>
+  );
+};
```

Comment:
you can do the same as suggested above for mobile filters also too much code is repeating